WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -13,6 +13,8 @@ final class OrdersTests: XCTestCase {
         try LoginFlow.login()
     }
 
+    let customerNote = "New customer note"
+
```

Comment:
Knowing that this is only used in the new test, WDYT of moving it inside the test?

Reply:
so i'm thinking that we could define all `Strings` near the top of the screen so in the case that have to update them, it's easier to find near the top of the screen (though this is not standard across all test files yet)